app-tasks/rf/src/rf/uploads/geotiff/create_bands.py

```diff
@@ -0,0 +1,42 @@
+import rasterio
```

Comment:
I added a TODO at the top of this file noting that this should be refactored once we have more information about what how other imaging systems work.